### PR TITLE
sign-in-thrid-party page ui added; button style added

### PR DESCRIPTION
**Pull request by jeesunikim**
Ticket: [[UI] Sign in to third party-services using Lyra](https://app.asana.com/0/1168666457741233/1168291088234545)
- Also added styles for all the buttons regarding its size (primary vs. small) and colors (warning vs. black vs. primary)

Preview:
* Overall Page
preview

* Buttons
primary-buttons
small-buttons

- There is a `prop` for `size`. **But** for colors, I didn't add props for. For `<GrantAccess/>`, we extended an existing style `Button` and created `RejectButton` styled component to style it with a black background

- I renamed `Button` in `src/popup/basics/index.tsx` to `BasicButton` since it's being used in a lot of places but I took `Button` for the all the styled buttons. Let me know if we want to change this to something else.

### CODE REVIEW

File: src/popup/Router.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import { POPUP_WIDTH } from "popup/constants";
 
 import Account from "popup/views/Account";
 import CreatePassword from "popup/views/CreatePassword";
-import GrantAccess from "popup/views/GrantAccess";
+import { GrantAccess } from "popup/views/GrantAccess";
```
Comment (jeesunikim) [inline src/popup/Router.tsx:23]: following product conventions

File: src/popup/basics/index.tsx [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
-import { COLOR_PALETTE } from "popup/styles";
 
-interface ErrorMessageProps {
-  error: string;
-}
+import { COLOR_PALETTE, FONT_WEIGHT } from "popup/styles";
 
-export const Button = styled.button`
+/* Button */
+export const BasicButton = styled.button`
```
Comment (jeesunikim) [inline src/popup/basics/index.tsx:8]: I changed its name to `BasicButton` and changed it in files that are using it.

File: src/popup/views/GrantAccess/index.tsx [hunk 3]
```diff
@@ -1,8 +1,43 @@
...
+  line-height: 1.9;
+  padding: 1.7rem 2rem;
+  margin: 0;
+`;
+const ButtonContainerEl = styled.div`
+  display: flex;
+  justify-content: space-around;
+  padding: 3rem 1.25rem;
+`;
+const RejectButton = styled(Button)`
```
Comment (jeesunikim) [inline src/popup/views/GrantAccess/index.tsx:36]: There is a `prop` for `size`. **But** for colors, I didn't add props for `<Button/>` so here I just extended an existing style `Button` and created `RejectButton`.

File: src/popup/components/Layout/Header/index.tsx [hunk 4]
```diff
@@ -32,8 +33,8 @@ type HeaderProps = {
   className?: string;
 };
 
-export const Header = ({ className }: HeaderProps) => (
-  <HeaderEl className={className}>
+export const Header = ({ className, ...props }: HeaderProps) => (
```
Comment (jeesunikim) [inline src/popup/components/Layout/Header/index.tsx:36]: Added `...props` to follow our [product conventions](https://github.com/stellar/product-conventions/blob/master/STYLEGUIDE.md#pass-props)

File: src/popup/views/GrantAccess/index.tsx [hunk 5]
```diff
@@ -1,8 +1,43 @@
...
+  color: ${COLOR_PALETTE.primary}};
+  font-weight: ${FONT_WEIGHT.light};
+  margin: 1rem 0 0.75rem;
+`;
+const H3 = styled.h3`
+  font-weight: ${FONT_WEIGHT.bold};
+  font-size: 0.95rem;
+  letter-spacing: 0.1px;
+  color: ${COLOR_PALETTE.primary}};
+`;
```
Comment (piyalbasu) [inline src/popup/views/GrantAccess/index.tsx:23]: some good feedback I got from Morley was to not name vars as their html tag, like `H1` or `H3`, but to name them something like `Header` and `Subheader` as the underlying html tag can change

Comment (jeesunikim) [reply src/popup/views/GrantAccess/index.tsx:23]: Updated it!

**Pull request merged**